source/js/payments-card.js

```diff
@@ -21,13 +21,26 @@ function setupBraintree() {
     );
 
   function showErrorMessage(msg) {
-    errorDiv.toggleAttribute("hidden", false);
-    errorDiv.innerHTML = msg;
+    if (errorDiv) {
```

Comment:
the error in Sentry was about the fact that we called `errorDiv.toggleAttribute("hidden", false);` while `errorDiv` was undefined. So this update makes sure to check whether it exists, and if not, put the error data on the console so it's at least "somewhere".